ui/app/src/services/content.ts

```diff
@@ -462,31 +462,37 @@ export function fetchItemsByContentType(
   if (typeof contentTypes === 'string') {
     contentTypes = [contentTypes];
   }
+
   return postJSON(`/studio/api/2/search/search.json?siteId=${site}`, {
     ...reversePluckProps(options, 'type'),
     filters: { 'content-type': contentTypes }
   }).pipe(
     map<AjaxResponse, { count: number; paths: string[] }>(({ response }) => ({
       count: response.result.total,
-      paths: response.result.items.filter((item) => item.type === options.type).map((item) => item.path)
+      paths: response.result.items.map((item) => item.path)
```

Comment:
Looks like you're dropping support for the type filter. why?
if we do it, then you need to drop it from the options of this method too.

Reply:
I was doing some testing and the result from the fetch is only including components, so looks like the filter is not doing anything, i think in the past the fetch result included groovy or something similar.